kiwi/builder/container.py

```diff
@@ -57,6 +60,23 @@ def __init__(self, xml_state, target_dir, root_dir):
         self.target_dir = target_dir
         self.container_config = xml_state.get_container_config()
         self.requested_container_type = xml_state.get_build_type_name()
+        self.base_image_uri = xml_state.get_derived_from_image_uri()
+        self.base_image = None
+
+        if self.base_image_uri:
+            base_image_origin = self.base_image_uri.translate()
+
+            # we expect the base image to be unpacked by the kiwi prepare step
+            # and stored inside of the root_dir/image directory
+            self.base_image = os.path.normpath(
+                os.sep.join(
+                    [
+                        root_dir, 'image',
+                        base_image_origin.replace('.tar.xz', '')
```

Comment:
I realized we do some path and file name assumptions here, and was a bit concerned if we are not creating unstable code under certain circumstances. Thoughts ?